internal/versions/versions.go

```diff
@@ -11,9 +9,8 @@ var (
 	// matches.
 	RegexHeaderVersionCutoff = semver.Version{Major: 2, Minor: 8}
 
-	// ExplicitRegexPathVersionCutoff is the Kong version prior to the addition of explicit "~" prefixes in regular
-	// expression paths.
-	ExplicitRegexPathVersionCutoff = semver.Version{Major: 3}
+	// ExplicitRegexPathVersionCutoff is the lowest Kong version adding the explicit "~" prefixes in regular expression paths.
+	ExplicitRegexPathVersionCutoff = semver.Version{Major: 3, Minor: 0}
```

Comment:
This seems to have been an error?

https://github.com/Kong/kong/blob/3.0.0/CHANGELOG.md#300 indicates that 3.0 is the version that introduces explicit "~" prefixes

Reply:
Isn't 
```
ExplicitRegexPathVersionCutoff = semver.Version{Major: 3, Minor: 0}
```
effectively the same as
```
ExplicitRegexPathVersionCutoff = semver.Version{Major: 3}
```
? In the original one, `Minor` would be zero as well. 🤔

Reply:
Yes but previously this was (the only? I'd need to double check) the version **prior** to X not **the lowest** supporting X.

Reply:
🤦 I see. Yeah, that makes sense. Must have been an error. 

Having a version that introduced the feature as a reference rather than the one prior to it seems more straightforward to me. 👍